NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -31,9 +31,45 @@
 using NUnit.Framework;
 using FirebirdSql.Data.FirebirdClient;
 using FirebirdSql.Data.Services;
+using System.Diagnostics;
 
 namespace FirebirdSql.Data.UnitTests
 {
+	[SetUpFixture]
+	public class FirstClass
+	{
+		[SetUp]
+		public void SetupDatabase()
+		{
+			Prepare(FbServerType.Default);
+			Prepare(FbServerType.Embedded);
```

Comment:
That was easy. :) https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/commit/263fe8c50c4c7c00c973ba3f71f2776db91ddb72